op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock

```diff
@@ -2,5 +2,8 @@ dependencies:
 - name: op-scim-bridge
   repository: https://1password.github.io/op-scim-helm/
   version: 2.6.0
-digest: sha256:9e4ff618b5e7241cf8d5765dd8c7fdb135f326fb4fabe9196000ae41befd7208
-generated: "2022-07-15T13:46:06.211713-07:00"
+- name: common
+  repository: https://charts.bitnami.com/bitnami
+  version: 1.16.1
```

Comment:
Is the idea behind this version pinning is to upgrade pessimistically?

Reply:
The Chart.lock file is generated by Helm when you build/update dependencies. We actually only pin the version to the latest available major version in the `Chart.yaml` (i.e. `version: ~1`).

So each time you run `helm dependency update ...` it will grab the latest available version that fits in that range (> 1.0.0 and < 2.0.0).